docs/program/python-venv.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Prepare your Python Virtual Environment"
-linkTitle: "Prepare Virtualenv"
+linkTitle: "Virtualenv for Python SDK"
```

Comment:
open to any other suggestions here but for now, think this might make it more clear these docs are just if you are using python SDK (not just using viam to program a robot at all)

Reply:
Kind of makes me wonder if this page should just be information in an expander on the program page

Reply:
I think it's a lot of info to put in an expander and people may look for it otherwise-- I like directing from requirements. Expander is not a bad idea though, so really up to you (if you want it I'll make the change!!)